draft-ietf-core-senml.md

```diff
@@ -544,18 +544,23 @@ humidity measurement.
 The CBOR {{RFC7049}} representation is equivalent to the JSON representation,
 with the following changes:
 
-* For JSON Numbers, the CBOR representation can use integers, floating 
-point numbers, or decimal fractions (CBOR Tag 4); however a 
-representation SHOULD be choosen such that when the CBOR value is 
-converted back to an IEEE double precision floating point value, it 
-has exactly the same value as the original Number.  For the version 
-number, only an unsigned integer is allowed. 
+* For JSON Numbers, the CBOR representation can use integers, floating
+point numbers, or decimal fractions (CBOR Tag 4); however a
+representation SHOULD be choosen such that when the CBOR value is
+converted back to an IEEE double precision floating point value, it
+has exactly the same value as the original Number.  For the version
+number, only an unsigned integer is allowed.
+
```

Comment:
There was no change in the above paragraph, just git somehow decided it changed since I had moved it.